src/constants/sites.ts

```diff
@@ -0,0 +1,22 @@
+export const TESTING_SITES = [
```

Comment:
Hmm why are we not going with the env var approach like SL and GGs is going with? Is there a long wait time before this feature gets rolled out to public?

Reply:
do those exist on frontend or backend? do note that this is supposed to roll out in phases, so for the most part, i don't think using an env var > this has any significant benefits

Reply:
sl in fe, gg in be 
nit would be to keep the approach same for all ff-ed features 
but ye, if you dont need dynamic reload then no high value

Reply:
resolving - not gonna do this because this is probably gonna be deleted by next sprint (2 weeks)